scipy/spatial/transform/rotation.py

```diff
@@ -1,12 +1,55 @@
 from __future__ import division, print_function, absolute_import
 
+__all__ = ['Rotation']
+
 import numpy as np
+import scipy.linalg
 
 class Rotation(object):
     """Rotation in 3 dimensions.
 
     This class will include initializers from different representations, converters
     and some useful algorithms such as quaternion slerp and rotation estimation.
+
     """
-    def __init__(self):
-        pass
+    def __init__(self, quaternions):
+        """Initialize class from possible normalized quaternions.
```

Comment:
We need to stick to some strategy for naming. I think perhaps use singular form (i.e. quaternion) and perhaps shorten it to `quat` for arguments. 

I'm also inclined to use `to_quat`, `as_quat`, but let's leave it for now.

Reply:
`quat` for arguments sounds fine.